src/python/pants/jvm/resolve/coursier_fetch_integration_test.py

```diff
@@ -170,6 +170,58 @@ def test_resolve_conflicting(rule_runner: RuleRunner) -> None:
         )
 
 
+@maybe_skip_jdk_test
+def test_resolve_with_broken_url(rule_runner: RuleRunner) -> None:
+
+    coordinate = Coordinate(
+        group="org.hamcrest",
+        artifact="hamcrest-core",
+        version="1.3_inexplicably_wrong",  # if the group/artifact/version is real, coursier will fallback
```

Comment:
Interesting. I think that that might suggest that it uses whatever pom it can find as well...?